### ISSUE TITLE
New website design

### ISSUE THREAD

**Issue by ngosang** [qbittorrent/qbittorrrent-website#5]
I will love a new design like this (but in blue color) => http://www.videolan.org/vlc/
We should open a new thread in the forum to request designs and designers. :)

**Comment by sledgehammer999** [comment]
No problem, but I dislike really long "endless scrolling ones". I would prefer news/changelog to at least have its own page. Not cram everything into one page.

**Comment by Chocobo1** [comment]
Might be possible to require the new design to use jekyll instead of PHP?
The website doesn't have dynamic content and it might be easier to blog in Markdown.

**Comment by ngosang** [comment]
There is no need to change php, if you don't have dynamic content there is only html in the php files...

**Comment by sledgehammer999** [comment]
No change please. We need a technology that is trivial to change hosting servers, or maybe go back to sf.net which supports php, perl, python, ruby, tcl.
I want to have options in case we need to migrate.

**Comment by TJBK** [comment]
Okay I'm messing around and so far i've got this:

![mess](https://my.mixtape.moe/ocqpoa.png)

Sorry it's not that good I have been doing it fast

**Comment by Chocobo1** [comment]
I hope the website also look good on mobile devices.

**Comment by Balls0fSteel** [comment]
Well you could also have two websites, one for desktop, one for mobile.
Mobile site should use AMP anyway, which requires a separate page.  
https://www.ampproject.org/

(When you Google a news story, and see the "AMP" text, tap it, it will instantly load. Google also puts these results in front. Okay ranking is not so important for the project, that's just a sidenote.)

**Comment by sledgehammer999** [comment]
FYI, I am not against the mockup @TJBK posted.

**Comment by TJBK** [comment]
Okay done a bit more work I'd like if someone could give me feedback 

Desktop:
![desktop](https://my.mixtape.moe/gocirg.png)

Mobile:
![mobile](https://my.mixtape.moe/bktzgo.png)

**Comment by Chocobo1** [comment]
@TJBK 
Cool, some thoughts following:
1. I was hoping a brighter theme, the deep blue is too dark & too plain IMO.
2. You could put more icons on it, e.g. for bitcoin, litecoin, maybe font awesome?
  Also see here for qbt image resources: https://github.com/qbittorrent/qBittorrent/tree/master/src/icons
3. The footer text "The qbittorrent project aims..." and bitcoin links should merge into 1 column, currently 2 columns doesn't feel balance for me.
4. The "support this project" button should align to middle (horizontally).
5. For the desktop layout, I'm using a monitor with res 1600x1050, at least I hope the "Home" page could fit in (with footer or not) so I don't need to scroll down to see the feature list. I guess the grey part on the top could shrink up a bit.

Thats all for now, thank you!

**Comment by sledgehammer999** [comment]
I agree with most of what @Chocobo1 said. I am not sure if I'll non-blue color theme. But who knows maybe, I'll do.
I have to add that at your 2nd post the "About qBittorrent" section looks weird with that white/gray color and the "stretching" of that box. However, maybe it will work better when render in the broswer instead of a screenshot.

**Comment by Balls0fSteel** [comment]
@Chocobo1 I think the entire mobile page could be crunched into an AMP page. What do you think? (I am just curious, I realize most hip n cool tech just dies - but boy, do I love instant loading pages on mobile.)

Well actually once you finish with the new site, I will try to convert/move it to AMP. Nothing should change.

**Comment by Chocobo1** [comment]
>Well actually once you finish with the new site, I will try to convert/move it to AMP. Nothing should change.

To be honest, I'm not too found of using google's AMP:
1. AMP might not work with other frameworks, e.g. bootstrap: http://stackoverflow.com/questions/38063272/using-accelerated-mobile-pages-with-bootstrap
2. I'm concerned that google might put analytics in it (not sure), I value that our current website doesn't contain any beacon that will identify users.

But don't get me wrong, I'm open to try, see how it works out before the final decision.

**Comment by Balls0fSteel** [comment]
Ah, don't worry about it. Once the site is done I will see if it's compatible. It's no big deal, really. 

I guess the most speedup comes from that companies are forced to build minimal sites. They have to throw away 5MB sites and come up with something clean and lean. And that's why it loads instantly on mobile.

**Comment by sledgehammer999** [comment]
>I'm concerned that google might put analytics in it (not sure), I value that our current website doesn't contain any beacon that will identify users.

Well, we have the various like buttons for that. In fact, there are studies/articles about facebook tracking even unregistered people via the like button code.
(in any case I run with **Ghostery** in block-all mode and I don't care about them)

**Comment by Balls0fSteel** [comment]
@sledgehammer999 Using uBlock Origin, you can add the Disconnect list too. Thus, you gain a very fast and lean adblocker, with privacy blocking as well. Just a tip.

If anyone uses Firefox, "tracking protection" is absolutely must have.
`privacy.trackingprotection.enabled` in `about:config`

(I know, sorry for "off-ing" in the ticket.)

**Comment by rovacado** [comment]
I decided to do my own mock-up, just because I haven't done any web-coding in a while and find it fun to relearn things. I used the PURE CSS framework for the base of the site as its small, extensible, and responsive. I have a live preview [Here](https://rovacado.000webhostapp.com/) (free webhost, no adds, just to host the mockup.) the site has 3 breakpoints or layouts, if you will. one for desktop, one for medium sized devices (such as tablets), and one for mobile devices. non of the links go anywhere, only the main page is mocked. I went with light colors, offwhites and such to be easy on the eyes, but also incoperated some of that classic blue that the current qB site has.

**Comment by Chocobo1** [comment]
@rovacado 
Nice mockup, some questions:
1. is it possible to use a horizontal navibar? Not sure it will be better than the current one... just wanted to see how it looks.
2. is it possible to replace the png icons by svg ones?

**Comment by sledgehammer999** [comment]
@rovacado personally I am not that fond of the new white theme. Also, personally, I would like the navbar horizontal for desktop use.

**Comment by worldsdream** [comment]
I can build the website. It will be mobile and desktop friendly. 

- homepage, modern
- news, it will be a blog posts with new versions and actual news if there is any. Also users can subscribe to rss.
- forum is a link to external
- download, modern
- screenshots, you can click easily on every screenshot (slider)
- wiki, development, bugs are links to external

What do you think @sledgehammer999 ?
